[Export] Support retraceability test on existing cases

andrewlee302 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #115413
* __->__ #115402

Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

    Ran 71 tests in 31.929s
    
    OK (expected failures=14)

andrewlee302 committed:
[Export] Support retraceability test on existing cases

Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

angelayi commented on test/export/test_export.py:
```diff
@@ -1532,7 +1548,9 @@ def forward(self, x):
 
+    @testing.expectedFailureRetraceability
     @testing.expectedFailureNonStrict
+    @testing.expectedFailureRetraceability
```
delete?

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 commented:
@pytorchbot merge

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 commented:
@pytorchbot merge

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

atalman commented:
@pytorchbot revert -m "OSSCI oncall, broke CI tests" -c nosignal

malfet commented:
It feels more like a landrace then no-signal (i.e. test_retraceability was added by 2nd PR on the stack, but probably it introduced some features which were not there to beging with...

andrewlee302 commented:
> Also, I have to say this is a suspicious test plan.

@malfet  I think the commit may get overridden by old summary. The local test is totally fine, like you mentioned

> On the other hand, I just run the test inside the container against commit base and it indeed passes:

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

FAILED (failures=6, errors=8, expected failures=7)

[ghstack-poisoned]

andrewlee302 commented:
Update the commit due to land race with another change.

@pytorchbot merge

andrewlee302 committed:
Update on "[Export] Support retraceability test on existing cases"


Summary:
Similar as #115399

Test Plan:
python test/export/test_retraceability.py

    Ran 71 tests in 31.929s
    
    OK (expected failures=14)

[ghstack-poisoned]
